eng/New-TestResources.ps1

```diff
@@ -55,6 +55,9 @@ param (
     [ValidateNotNullOrEmpty()]
     [hashtable] $AdditionalParameters,
```

Comment:
What is our longer term plan to keep these scripts updated across repos? Do we have an issue tracking that?

Reply:
This is something that @heaths brought up and we've talked about but haven't reached a conclusion. I've opened https://github.com/Azure/azure-sdk/issues/934 to track this.